app/helpers/friendships_helper.rb

```diff
@@ -46,4 +23,35 @@ def get_friendship(user)
     relation = Friendship.find_by(sender:user.id, receiver:current_user.id) || 
```

Comment:
Not supposed to have application logic in your helper files. `find_by` methods are application logic related.